common/notification/hooks.ts

```diff
@@ -17,6 +21,27 @@ registerStudentHook('cancel-remission-request', 'Cancels the remission request(s
     await cancelRemissionRequest(student);
 });
 
+registerStudentHook(
+    'attempt-invite-instructor-to-reflection-meeting',
+    'Instructors are invited to participate in a reflection meeting after their very first group appointment',
+    async (student, context) => {
+        if (student.isStudent) {
```

Comment:
is it right that we don't send the invitation if the instructor is a tutor in addition to being an instructor?

Reply:
Yeah, I think that's not fully correct. The requested condition was.
> HuH, die nur Kurse geben und kein Match haben

I adjusted the condition to just verify that the user doesn't have an active match.

Thanks!